cylc/rose/entry_points.py

```diff
@@ -54,12 +55,12 @@ def __str__(self):
         return msg
 
 
-def pre_configure(srcdir=None, opts=None, rundir=None):
+def pre_configure(srcdir, opts, rundir=None):
```

Comment:
I have just spotted, the `pre_configure` entry point does not actually provide the `rundir` arg

Reply:
No, but there's no reason someone else might not use the interface.

Reply:
```suggestion
def pre_configure(srcdir, opts):
```